src/eko/__init__.py

```diff
@@ -18,14 +25,12 @@ def run_dglap(theory_card, operators_card):
 
     Parameters
     ----------
-        setup : dict
-            input card - see :doc:`/code/IO`
+    setup : dict
+        input card - see :doc:`/code/IO`
 
     Returns
     -------
-        output : dict
-            output dictionary - see :doc:`/code/IO`
+    output.EKO
+        output object - see :doc:`/code/IO`
```

Comment:
`output object` is too generic, you can also leave it empty.

Please replace with something like "computed operator" - same length, but more explicit. If possible expand a bit.